### ISSUE TITLE
LOGIN ISSUE!! Please Help

### ISSUE THREAD

**Issue by pogodpi** [finndev/PokeBuddy#457]
I couldn't login into the pokebuddy with "FREE-BETA" login.. it says "login failed".. I tried everything.. But still it fails.. Someone help me resolve tis issue!!
![capture](https://cloud.githubusercontent.com/assets/20889930/17470323/e88e9780-5d58-11e6-8a37-a6f197d69533.JPG)

**Comment by eyegood38** [comment]
Try a PC restart?

**Comment by pogodpi** [comment]
Tried .. downloaded the latest version too.. extracted the pokebuddy to the master and tried logging in.. sync'ed the time too..  i tried almost everything.. but still it fails..

**Comment by eyegood38** [comment]
no anti virus to turn off?

**Comment by pogodpi** [comment]
yeah turned it off too.. but no use..

**Comment by eyegood38** [comment]
Got another PC to try?

**Comment by eyegood38** [comment]
Also, the AV might have deleted something. Did you turn it off before unpacking and opening? Maybe retry download and extract with AV off?

**Comment by curtiola** [comment]
Maybe it's windows defender's firewall? If your Antivirus doesn't override it that is..

**Comment by pogodpi** [comment]
No other personal PCs right now.. my AV s turned off from the beginning.. but no use  :(

**Comment by pogodpi** [comment]
ok will try turning off everything and try again..

**Comment by pogodpi** [comment]
No use.. tried everything again.. Can't login :( I'm the only one facing this problem  :(

**Comment by curtiola** [comment]
Yeah its strange, as dumb of a question as it is.. did you try run as admin?

**Comment by pogodpi** [comment]
Yes i did.. running the latest version.. tried pc restart.. AV n firewall turned off.. running as admin.. inside the master.. no use..

**Comment by curtiola** [comment]
Huh, my knowledge is limited lol sorry I couldn't be of any help
